space_age/app/views/users/show.html.erb

```diff
@@ -1,25 +1,40 @@
 <br>
 <div class="row small-6 offset-3">
-  <h4 class="centered-text">Welcome to Your Dashboard, <%= @user.username %></h4>
+  <h4 class="centered-text">Welcome to Your Dashboard, <%= @user.username.capitalize %></h4>
   <h4 class="centered-text"><%= @user.email %></h4>
```

Comment:
we can still make this happen. 
class User
 def formatted_name
   username.upcase
 end
end